### PR TITLE
Docs: Add AbstractUser import to custom user model

### PR DESCRIPTION
**Pull request by lb-**
Add AbstractUser import to example `from django.contrib.auth.models import AbstractUser`

### CODE REVIEW

File: docs/advanced_topics/customisation/custom_user_models.rst [hunk 1]
```diff
@@ -11,7 +11,8 @@ Create a custom user model. In this case we extend the ``AbstractUser`` class an
 two fields. The foreign key references another model (not shown).
 
 .. code-block:: python
-
+  from django.contrib.auth.models import AbstractUser
```
Comment (gasman) [inline docs/advanced_topics/customisation/custom_user_models.rst]: Need an extra blank line above this to be valid RestructuredText syntax.

Comment (lb-) [reply docs/advanced_topics/customisation/custom_user_models.rst]: Thanks

**Comment by lb-** [comment]
@gasman - let me know if you are happy for me to merge this one in

**Comment by gasman** [comment]
@lb- Yep, go for it!

**Comment by lb-** [comment]
Merged in 03cd4056df162fce60109ed01edaf02d755af598

**Pull request closed without merging**